website/docs/r/portal_tenant_configuration.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Portal"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_portal_tenant_configuration"
+description: |-
+  Manages Portal Tenant Configuration.
+---
+
+# azurerm_portal_tenant_configuration
+
+Manages Portal Tenant Configuration.
+
+~> **Note:** User has to be a Tenant Admin for managing this resource.
```

Comment:
The requirement here is Owner or Contributor at scope '"/"', Owner is not mandatory. We should update the instructions to the lower permissions based on principal of least privilege.

Reply:
Updated